README.rst

```diff
@@ -60,6 +60,49 @@ or:
 
     openstack baremetal node unset $NODE_ID --extra disable_bios_version_check
 
+system_nic
+-----------
+
+Provides a hardware manager to verify the firmware version of network interface
+cards against a list of expected versions in the Ironic node info. The following
+node info is expected to be set:
+
+.. code-block::
+
+  "extra": {
+    "nic_firmware": [
+      {
+        "vendor_id": "<vendor_id>",
+        "device_id": "<device_id>",
+        "firmware_version": "<firmware_version>"
```

Comment:
Should we make sure that this data is collected by inspection?

Reply:
I still don't think that we should be automatically applying information found via inspection to the verification rules - it could open up a security hole.

Reply:
Agreed that it shouldn't be automatic. I was just thinking that a user could manually look at the inspection data to get an idea of what the dict should look like. They'd need to manually validate it / set which BIOS version they actually wanted.